config/packages/sulu_security.yaml

```diff
@@ -0,0 +1,9 @@
+sulu_security:
+    password_policy:
+        enabled: false
+
+# By default sulu provides a simple password_policy pattern of ".{8,}"
+# which can be made more complex with the following lines.
```

Comment:
```suggestion
# Sulu uses the simple password_policy pattern ".{8,}" by default
# You can change it to a more complex pattern with the following lines:
```

Reply:
if we make the default value explicit like @alexander-schranz described above we dont need this comment 🙂